content/departments/support/process/removing-repositores-from-cloud.md

```diff
@@ -0,0 +1,113 @@
+# Removing repositores from cloud
+
+This document is a guide on how to completely remove repositories from Sourcegraph cloud.
+
+## Prerequisites
+
+The user should have privileged access to be able to perform this on Sourcegraph cloud.
+
+⚠️ WARNING: Be careful to make sure everything checks out as per the removal request.
+
+## Process of removing repositories
+
+We consider the following kinds of repositories:
+
+- Public forks of private repositories
+- Public repositories that are made private
+- Public repositories added by other users or organizations other than the owner of the repository
+- Private repositories
+
+The removal process is the same across all the visibility types listed above.
+
+Depending on the environment, exec into `pgsql` with:
+
+**Kubernetes**
+
+```
+kubectl exec -ti $PGSQL_POD -- psql -U sg
+```
+
+**Docker**
+
+```
+docker exec -it $PGSQL_CONTAINER -- psql -U sg
+```
+
+**Option 1 ('Hard' delete)**
+
+A hard delete leaves no trace of the repository in the database
+
+1. If you know the name of the repo by it's URI get the id:
+
+```
+SELECT id FROM repo WHERE name LIKE 'example.com/example/repo';
+```
+
+or
+
+```
+SELECT id FROM repo WHERE uri LIKE 'example.com/example/repo';
+```
```

Comment:
Are the queries exchanged in position since the point states the query is for lookup by URI?

Reply:
Got it to make sense with the new update. Referencing the URI only